ambry-store/src/main/java/com.github.ambry.store/PersistentIndex.java

```diff
@@ -833,9 +861,9 @@ private StoreFindToken findEntriesFromSegmentStartOffset(Offset initialSegmentSt
     } else {
       // if newTokenSegmentStartOffset is set, then we did fetch entries from that segment, otherwise return an
       // uninitialized token
-      return newTokenSegmentStartOffset == null ? new StoreFindToken()
+      return newTokenSegmentStartOffset == null ? new StoreFindToken(incarnationId)
```

Comment:
I thought we agreed to remove incarnation Id from an uninitialized token. Did something change?